_includes/cockroachcloud/private-preview.md

```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-This feature is a Private Preview release and can be enabled only on clusters in enrolled CockroachDB Cloud organizations. To enroll your organization in the private preview, contact your Cockroach Labs account team.
+This feature is a Preview release and can be enabled only on clusters in enrolled CockroachDB Cloud organizations. To enroll your organization in the private preview, contact your Cockroach Labs account team.
```

Comment:
Same comment about preview spelling here. Also why does it say private preview?

Reply:
I just missed the second "private". I've fixed it now, and I've lowercased the first instance of "preview". Otherwise we are stuck needing to capitalize it if we mention it in prose, and I don't think we want that.

Reply:
I agree with your call Matt.